domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/service/EntityToResponseDTOMapper.java

```diff
@@ -109,7 +110,14 @@
     CaseField map(CaseFieldEntity caseFieldEntity);
 
     @Mapping(source = "fieldTypeEntity.reference", target = "id")
-    @Mapping(source = "fieldTypeEntity.listItems", target = "fixedListItems")
+    @Mapping(
+        expression = "java("
+                     + "           uk.gov.hmcts.ccd.definition.store.domain.service.EntityToResponseDTOMapper.FieldTypeEntityMapper.map("
```

Comment:
Can you revert to the original name FixedListMapper please?

Reply:
DONE